src/extension.ts

```diff
@@ -785,6 +791,24 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
     vscode.commands.registerCommand("vscode-objectscript.compileWithFlags", () => importAndCompile(true)),
     vscode.commands.registerCommand("vscode-objectscript.compileAll", () => namespaceCompile(false)),
     vscode.commands.registerCommand("vscode-objectscript.compileAllWithFlags", () => namespaceCompile(true)),
+    vscode.commands.registerCommand("vscode-objectscript.refreshLocalFile", async (_file, files) => {
```

Comment:
Since this command is only in the command palette now the implementation should be really simple, perhaps only a few lines:

```ts
const file = currentFile();
if (!file) {
  return;
}
loadChanges([file]);
```

IMO you also don't need to special Output channel messages anymore since you're only refreshing one file.